tests/unit/ClientTest.php

```diff
@@ -144,11 +145,18 @@ public function testFactoryMethod()
 
     public function testTimeoutIsFloat()
     {
-        $client =  $this->getClient('test', 'test', false, 0.5);
+        $client =  $this->getClient('test', 'test', false, false, 0.5);
 
         $this->assertEquals(0.5, $client->getTimeout());
     }
 
+    public function testVerifySSLIsBoolean()
```

Comment:
Can you enforce this assertion to `testVerifySSLIsTrue`? I know that other tests are like your but you know :) Make it better is good..

Thanks a lot